🚨Do not merge until Oct 18 launch - IDE GA docs (#2152)
fishtown-analytics/docs.getdbt.com: website/docs/community/resources/long-lived-discussions-guidelines.md

---
title: "Community Forum guidelines"
id: "forum-guidelines"
---

## What is the dbt Community Forum?

> [The dbt Community Forum](https://discourse.getdbt.com) is the platform for getting support with dbt as well as to have long-lived discussions about dbt, analytics engineering, and analytics. It's a place for us to build up a long-lasting knowledge base around the common challenges, opportunities, and patterns we work with every day. The forum is different from the dbt Community Slack in a few key ways, most importantly it is:

Reviewer: ```suggestion
[The dbt Community Forum](https://discourse.getdbt.com) is the platform for getting support with dbt as well as to have long-lived discussions about dbt, analytics engineering, and analytics. It's a place for all of us to build a long-lasting knowledge base around the common challenges, opportunities, and patterns we work with every day. The forum is different from the dbt Community Slack in a few key ways, most importantly it is:
```

> - **Asynchronous** and **long-lived** - sometimes conversations continue over weeks, months, and beyond.

Reviewer: ```suggestion
- **Asynchronous** and **long-lived** &mdash; sometimes conversations continue over weeks, months, and beyond.
```

> - **Intentional** - we recommend taking at least 5 to 10 minutes thinking about and shaping your initial post and any comments.

Reviewer: ```suggestion
- **Intentional** &mdash; we recommend taking at least 5 to 10 minutes thinking about and shaping your initial post and any comments.
```

> - **Citable** - Slack conversations tend to be great in the moment but get lost in the flow — forum posts can more easily shared and referenced.

Reviewer: ```suggestion
- **Citable** &mdash; Slack conversations tend to be great in the moment but get lost in the flow. Forum posts are more easily shared and referenced.
```

**Guidelines for engaging on the Forum**

The community [Rules of the Road](https://docs.getdbt.com/docs/contributing/slack-rules-of-the-road) apply, and following them is the best way to get momentum behind your discussion or answers to your questions. The following guidelines will set you up for success:

- Be respectful
- Put effort into your posts
- Mark replies as Solutions in the Help section
- Don’t double post

## Categories

The forum is broken down into three categories:

- [Help](https://discourse.getdbt.com/c/help/19)
  - This is a Q&A style forum where you can ask the dbt Community for help with specific questions about dbt, dbt Cloud, data modeling, or anything else you want a definitive answer on.
  - This category is for questions which can plausibly have a _single correct answer_.
    - ✅ How do I debug this Jinja error?
    - ✅ How do I set up CI in dbt Cloud on GitHub?

>     - ❌ What is the best way to do marketing attribution? (More general Discussions like this are perfect for the [In-Depth Discussions](https://discourse.getdbt.com/c/discussions/21) category)

Reviewer: ```suggestion
    - ❌ What is the best way to do marketing attribution? More general Discussions like this are perfect for the [In-Depth Discussions](https://discourse.getdbt.com/c/discussions/21) category
```

- [Show and Tell](https://discourse.getdbt.com/c/show-and-tell/22)

>   - This is the place to show off all of the cool things you are doing in dbt. Whether it’s a new macro, design pattern, or package, post here to show the community what you are up to!

Reviewer: ```suggestion
  - This is the place to show off all the cool things you are doing in dbt. Whether it’s a new macro, design pattern, or package, post here to show the community what you are up to!
```

- [In-Depth Discussions](https://discourse.getdbt.com/c/discussions/21)
  - Share anything you’re thinking about that has to do with dbt or analytics engineering! This is a great place to jot down some thoughts to share with the community or spark a discussion on a topic that’s currently interesting you.

## Inclusivity on the Community Forum

> We are **strongly** committed to building a community where everyone can feel welcome.

Reviewer: ```suggestion
We are _strongly_ committed to building a community where everyone can feel welcome.
```

Reviewer: per style guide, use italics for emphasis (not bold)

> The dbt community started with people who were not traditionally considered “technical”, did not have ownership over technical systems, and were often left out of organizational decision-making. We came together to learn from each other, solve hard problems together, and to help build a new discipline where data folks have greater ownership over our own work. It really matters to us that everyone feels like they can ask questions and engage, no matter their professional or personal background.

Reviewer: ```suggestion
The dbt community started with people who weren't traditionally considered “technical”, didn't have ownership over technical systems, and were often left out of organizational decision-making. We came together to learn from each other, solve hard problems together, and to help build a new discipline where data folks have greater ownership over our own work. It really matters to us that everyone feels like they can ask questions and engage, no matter their professional or personal background.
```

> Technical forums have the potential to replicate harmful power structures, and can feel intimidating or hostile. We are working hard to create and sustain an inclusive environment through community-building, technological solutions, inclusive content, and diverse contributors. This is a long-term project, and we will continue to iterate and make improvements.

Reviewer: ```suggestion
Technical forums have the potential to replicate harmful power structures, and can feel intimidating or hostile. We're working hard to create and sustain an inclusive environment through community-building, technological solutions, inclusive content, and diverse contributors. This is a long-term project, and we'll continue to iterate and make improvements.
```

> If you have any ideas or feedback on how to make this space friendlier or more inclusive please let us know, either on the community Slack in the #community-strategy channel or via email at [community@dbtlabs.com](mailto:community@dbtlabs.com). We want to hear from you!

Reviewer: ```suggestion
If you have any ideas or feedback on how to make this space friendlier or more inclusive please let us know, either on the community Slack in the [#community-strategy](https://getdbt.slack.com/archives/C0171LNRS6S) channel or with [email](mailto:community@dbtlabs.com). We want to hear from you! 
```

> ## Following new and ongoing Discussions

Reviewer: ```suggestion
## Following new and ongoing discussions
```

The best way to stay up to date is to [browse the forum](https://discourse.getdbt.com/) directly. You can also Track or Watch specific threads or the whole category to receive updates on them without commenting.

> Each category also has a companion Slack channel (#advice-dbt-for-beginners, #show-and-tell and #in-depth-discussions). You can reply to the initial post in Slack and it will be added as a comment on the forum thread, allowing you to participate from inside Slack if you prefer.

Reviewer: ```suggestion
Each category also has a companion Slack channel ([#advice-dbt-for-beginners](https://getdbt.slack.com/archives/CBSQTAPLG), [#i-made-this](https://getdbt.slack.com/archives/C01NH3F2E05) and #in-depth-discussions). You can reply to the initial post in Slack and it will be added as a comment on the forum thread, allowing you to participate from inside Slack if you prefer.
```

Reviewer: trying to embed slack links. i couldn't find the channel "#in-depth-discussions". was it renamed? 

#show-and-tell was renamed to "#i-made-this"
